### ISSUE TITLE
Trouble building envoy using bazel

### ISSUE THREAD

**Issue by achasveachas** [envoyproxy/envoy#7250]
**Issue Template**

*Title*: *Trouble building envoy using bazel*

*Description*:
We are trying to build the master branch of envoy using Bazel version 0.22.0 but we seem to have gotten ourselves into a bad state.

We decided to start fresh, we downloaded a fresh Bazel executable and tried following the [documentation](https://github.com/envoyproxy/envoy/blob/master/bazel/README.md) on building envoy but we're running into the following error:
```
$ bazel build //source/...
WARNING: The following rc files are no longer being read, please transfer their contents or import their path into one of the standard rc files:
/home/pivotal/workspace/vmfiles/vmshare/envoy/envoy/tools/bazel.rc
Extracting Bazel installation...
Starting local Bazel server and connecting to it...
INFO: Invocation ID: af52f49e-922e-4b0a-8bd9-00919eca76a0
ERROR: error loading package 'source/server/http': in /home/pivotal/workspace/vmfiles/vmshare/envoy/envoy/bazel/envoy_build_system.bzl: in /home/pivotal/workspace/vmfiles/vmshare/envoy/envoy/bazel/envoy_library.bzl: in /home/pivotal/.cache/bazel/_bazel_pivotal/c54538fa8ade9fd72e892f1d4bbf0793/external/envoy_api/bazel/api_build_system.bzl: in /home/pivotal/.cache/bazel/_bazel_pivotal/c54538fa8ade9fd72e892f1d4bbf0793/external/com_envoyproxy_protoc_gen_validate/bazel/pgv_proto_library.bzl: in /home/pivotal/.cache/bazel/_bazel_pivotal/c54538fa8ade9fd72e892f1d4bbf0793/external/com_envoyproxy_protoc_gen_validate/bazel/protobuf.bzl: Unable to load file '@bazel_tools//tools/jdk:toolchain_utils.bzl': file doesn't exist
INFO: Elapsed time: 9.156s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
    currently loading: source/server ... (138 packages)
```
Searching through past issues, we weren't able to find a solution, any ideas how we can debug this?

CC: @sophiewigmore @yaelharel

**Comment by moderation** [comment]
@achasveachas You should try with the current version of Bazel which is 0.26.1 - https://github.com/bazelbuild/bazel/releases/tag/0.26.1. I presume you are on Linux? After getting the latest Bazel you might want to clean up with `bazel clean --expunge` and try again.

**Comment by achasveachas** [comment]
Yes, we tried 0.26.1 but ran into the same issues with that version as well (and 0.24.0).

We saw a few people mentioning that 0.26 seemed to still have some bugs which is why we rolled back to 22 which we knew wokerd for us until recently.

**Comment by moderation** [comment]
- Linux?
- Build command? Is it `bazel build //source/exe:envoy-static.stripped` or something similar?
- `/home/pivotal/workspace/vmfiles/vmshare/` - Are you building on a remote file system?
I'm building on Linux (Red Hat 7), Ubuntu 18.04 with no issues and have been for a while. Will need more detail in order to help.

**Comment by achasveachas** [comment]
- We're on Linux 18.04 as well.
- The command we're using is just `bazel build //source/...`, but ran into the same issues when trying to build `bazel build //source/exe:envoy-static`
- The directory is also mounted into a Windows VM (which is why it's under our `vmshare` directory, but we're running the build on the Linux host.

We were able to run everything just fine until recently, it's just that we had to overhaul our dev environment recently and weren't able to get it working again. We have a feeling it might be some simple fix like a missing dependency or something, but we can't figure it out.
